browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -707,15 +707,18 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			if (data.children[0].text && data.children[0].text !== '') {
 				var expander = L.DomUtil.create('div', 'ui-expander ' + builder.options.cssClass, container);
 				expander.tabIndex = '0';
+				expander.setAttribute('aria-expanded', false);
 				var label = L.DomUtil.create('span', 'ui-expander-label ' + builder.options.cssClass, expander);
 				label.innerText = builder._cleanText(data.children[0].text);
 				label.id = data.children[0].id ? data.children[0].id : data.id + '-label';
 				if (data.children[0].visible === false)
 					L.DomUtil.addClass(label, 'hidden');
 				builder.postProcess(expander, data.children[0]);
 
-				if (data.children.length > 1 && expanded)
-					$(label).addClass('expanded');
+				if (data.children.length > 1 && expanded){
+					label.classList.add('expanded');
+					expander.setAttribute('aria-expanded', true);
+				}
```

Comment:
this section is duplicated, we have `if (expanded) ` below: https://github.com/CollaboraOnline/online/pull/11655/files#diff-1951654bafeb4ed39d89489812ff8da269c905feabbe179ccfd312d1f2d19fb6R747

could we do it once in single place?